rfc/README.md

```diff
@@ -112,4 +112,5 @@ The list of all RFCs can be found here.
 | 74         | [`HoodieStorage`: Hudi Storage Abstraction and APIs](./rfc-74/rfc-74.md)                                                                                                                                             | `UNDER REVIEW` |
 | 75         | [Hudi-Native HFile Reader and Writer](./rfc-75/rfc-75.md)                                                                                                                                                            | `UNDER REVIEW` |
 | 76         | [Auto Record key generation](./rfc-76/rfc-76.md)                                                                                                                                                                     | `IN PROGRESS`  |
-| 77         | [Secondary Index](./rfc-77/rfc-77.md)                                                                                                                                                                                | `UNDER REVIEW` |
\ No newline at end of file
+| 77         | [Secondary Index](./rfc-77/rfc-77.md)                                                                                                                                                                                | `UNDER REVIEW` |
+| 78         | [Concurrent schema evolution detection](./rfc-78/rfc-78.md)                                                                                                                                                                                | `ONGOING` |
```

Comment:
Put up a separate PR for claiming the RFC number and entry: https://hudi.apache.org/contribute/rfc-process#proposing-the-rfc.

The `README.md` is outdated.  You need to pull the latest.